Game crashes on load due to CrashReportManager

`CrashReportManager.generateDetails()` is called in `CrashReportMixin`, which mixes into a method that fires before minecraft loads. `CrashReportManager` references the `Managers` class, causing it to call all the constructors for its static manager variables - but many of those managers rely on the minecraft instance being active, and cause the game to crash when called before that's the case.

```
Caused by: java.lang.NullPointerException: Cannot read field "gameDirectory" because the return value of "com.wynntils.mc.utils.McUtils.mc()" is null
	at com.wynntils.core.WynntilsMod.<clinit>(WynntilsMod.java:34) ~[main/:?]
	at com.wynntils.core.net.NetManager.<clinit>(NetManager.java:28) ~[main/:?]
	at com.wynntils.core.net.UrlManager.<init>(UrlManager.java:36) ~[main/:?]
	at com.wynntils.core.managers.Managers.<clinit>(Managers.java:33) ~[main/:?]
	at com.wynntils.core.mod.CrashReportManager.generateDetails(CrashReportManager.java:29) ~[main/:?]
	at net.minecraft.CrashReport.handler$zzj000$addWynntilsDetails(CrashReport.java:525) ~[minecraft-project-@fabric-clientOnly-named.jar:?]
	at net.minecraft.CrashReport.getDetails(CrashReport.java:77) ~[minecraft-project-@fabric-clientOnly-named.jar:?]
	at net.minecraft.CrashReport.getFriendlyReport(CrashReport.java:132) ~[minecraft-project-@fabric-clientOnly-named.jar:?]
	at net.minecraft.CrashReport.preload(CrashReport.java:272) ~[minecraft-project-@fabric-clientOnly-named.jar:?]
	at net.minecraft.client.main.Main.main(Main.java:151) ~[minecraft-project-@fabric-clientOnly-named.jar:?]
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461) ~[fabric-loader-0.14.10.jar:?]
	... 4 more
```

--- a/common/src/main/java/com/wynntils/core/mod/CrashReportManager.java
+++ b/common/src/main/java/com/wynntils/core/mod/CrashReportManager.java
@@ -6,6 +6,7 @@ package com.wynntils.core.mod;
 
 import com.wynntils.core.managers.Manager;
 import com.wynntils.core.managers.Managers;
+import com.wynntils.mc.utils.McUtils;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
@@ -26,7 +27,7 @@ public final class CrashReportManager extends Manager {
     public static CrashReportCategory generateDetails() {
         CrashReportCategory wynntilsCategory = new CrashReportCategory("Wynntils");
 
-        if (Managers.CrashReport == null) {
+        if (McUtils.mc() == null) {
             wynntilsCategory.setDetail("No crash handler loaded yet", "");
             return wynntilsCategory;
         }

